File: Algorithm/programmers/Level2/JadenCase 문자열 만들기.py
def solution(s):
    s = s.split(' ')
    answer = []
    for char in s:
        tmp = ''
        for i in range(len(char)):
            if i == 0:
                tmp += char[i].upper()
            else:
                tmp += char[i].lower()
        answer.append(tmp)

    return ' '.join(answer)

File: Algorithm/programmers/Level2/JadenCase 문자열 만들기.js
function solution(s) {
  let arr = s.split(" ");
  let answer = [];

  for (let str of arr) {
    str = str.toLowerCase();
    if (str) {
      str = str[0].toUpperCase() + str.slice(1);
    }
    answer.push(str);
  }

  return answer.join(" ");
}

function solution(s) {
  return s
    .split(" ")
    .map((v) => v.charAt(0).toUpperCase() + v.substring(1).toLowerCase())
    .join(" ");
}
